R/globals.R

```diff
@@ -0,0 +1,3 @@
+library(GAMBLR.data)
```

Comment:
Maybe, my previous comments about GAMBLR.data being loaded in the examples is taken care of with this? If so, disregard my previous comment about examples.

Reply:
I was planning to add here some explanation about what this `global.R` file does, but I forgot. 

`all_functions <- as.vector( lsf.str("package:GAMBLR.data") )` get all function names from GAMBLR.data. But for this work, we need `library(GAMBLR.data)` before it. 

`utils::globalVariables(all_functions)` marks all these functions to not return notes like 
```
fancy_propcov_plot: no visible global function definition for
  ‘collate_qc_results’
```
when `devtools::check()`` is called. I did this because we are not using prefixes of importFrom when the code uses a function from GAMBLR.data (giving the flexibility to choose GAMBLR.data or GAMBLR.results as the source).